File: 0136-single-number/0136-single-number.java
class Solution {
    public int singleNumber(int[] nums) {
        int n = nums.length;
        if(n == 1){
            return nums[0];
        }
        
        for(int i=0;i<n;i++){
            int count = 0;
            for(int j=0;j<n;j++){
                if(i!=j && nums[i] == nums[j]){
                    count ++;
                    break;
                }
            }
            if(count== 0){
                return nums[i];
            }
        }
        return -1;
    }
}

File: 0136-single-number/0136-single-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var singleNumber = function(nums) {
    let count = 0;
    for(let i = 0;i<nums.length;i++){
        for(let j = 0;j<nums.length;j++){
            if(i != j){
                if(nums[i] == nums[j]){
                    count++;
                }
            }
        }
        if(count >= 1){
            count = 0;
        }else{
            return nums[i];
        }
    }
    return -1;
};